test/mocha/validator.spec.js

```diff
@@ -0,0 +1,42 @@
+var assert = require('assert')
+import { domainFormat } from '../../src/renderer/utils/validator'
```

Comment:
__[ESLint]__

Error - Import in body of module; reorder to top. (import/first)


<!-- Comment by [Sider](https://sider.review) -->